pkg/sink/codec/encoder_group.go

```diff
@@ -70,15 +76,40 @@ func NewEncoderGroup(builder RowEventEncoderBuilder,
 	for i := 0; i < concurrency; i++ {
 		inputCh[i] = make(chan *future, defaultInputChanSize)
 	}
+	outCh := make(chan *future, defaultInputChanSize*concurrency)
+
+	var bootstrapWorker *bootstrapWorker
+	protocol := util.GetOrZero(cfg.Protocol)
+	if protocol == config.ProtocolSimple.String() {
+		log.Info("Sending bootstrap event is enable for simple protocol, creating bootstrap worker...",
+			zap.Stringer("changefeed", changefeedID))
+		sendBootstrapIntervalInSec := util.GetOrZero(cfg.SendBootstrapIntervalInSec)
```

Comment:
`sendBootstrapIntervalInSec` should already be correctly initialized when create the changefeed, there is no need to adjust the value.